server/src/index.ts

```diff
@@ -47,9 +51,35 @@ app.use((err: any, req: Request, res: Response, next: NextFunction) => {
   res.render('error');
 });
 
-const port = process.env.PORT || 5000; // Use environment variable or default to 3001
-app.listen(port, () => {
-  console.log(`Server running on http://localhost:${port}`);
+// const port = process.env.PORT || 3001; // Use environment variable or default to 3001
+// app.listen(port, () => {
+//   console.log(`Server running on http://localhost:${port}`);
+// });
```

Comment:
Please don't leave commented out code in your final solution. Remove the code that's not needed.

Reply:
yes, thank you. This code smells, I haven't taken it under consideration while pushing the code, It was before our recent class. Thanks for the update The next pr will not contain such activity.